life/original/19-chapter-19.adoc

```diff
@@ -1771,19 +1840,20 @@ and separate from all your false worships by his own light, Spirit and truth,
 and to follow his dear Son in the way of regeneration,
 whose love to me hath taught me to love all mankind, and to seek their salvation.
 
+[.signed-section-signature]
 William Penn.
 
+[.signed-section-context-close]
 Amsterdam, the 20th of tho Eighth month, 1677.
 
 [.asterism]
 '''
 
```

Comment:
The linter says that there are two blank lines here. You'll have to kill one of them 👍🏻